pkg/controllers/clusterresource/keymanagementprovider_controller.go

```diff
@@ -20,20 +20,18 @@ import (
 	"context"
 	"encoding/json"
 	"fmt"
-	"maps"
 
 	"github.com/ratify-project/ratify/internal/constants"
 	_ "github.com/ratify-project/ratify/pkg/keymanagementprovider/azurekeyvault" // register azure key vault key management provider
 	_ "github.com/ratify-project/ratify/pkg/keymanagementprovider/inline"        // register inline key management provider
-	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	"github.com/ratify-project/ratify/pkg/keymanagementprovider/refresh"         // register inline key management provider
```

Comment:
nit: comment is not correct?

Reply:
You're correct. It's not correct. :) Copilot has failed me. I'll get it updated.